### ISSUE TITLE
Document Xcode project failing because of CodeSign

### ISSUE THREAD

**Issue by allejo** [BZFlag-Dev/bzflag#297]
If you run into this problem when building locally

```
/Users/allejo/Library/Developer/Xcode/DerivedData/BZFlag-uuid/Build/Products/Debug/BZFlag-2.4.23.app: code object is not signed at all
In subcomponent: /Users/allejo/Library/Developer/Xcode/DerivedData/BZFlag-uuid/Build/Products/Debug/BZFlag-2.4.23.app/Contents/Frameworks/SDL2.framework
Command CodeSign failed with a nonzero exit code
```

here's _one_ way of fix it

1. Go to "Build Settings" for the main BZ target
   ![image](https://user-images.githubusercontent.com/1246453/146310071-f4233a01-e69d-4cb6-a8ed-3fb0d2ea192c.png)
2. Select "Code Signing Identity" and choose "Other"
   ![image](https://user-images.githubusercontent.com/1246453/146310131-685cf925-9c04-4b24-9a56-90648e920437.png)
3. Erase the contents so that it's empty and hit Enter. This'll remove the value from the "Code Signing Identity" field and let you build normally for local
   ![image](https://user-images.githubusercontent.com/1246453/146310187-dcdf96c6-a944-48aa-ae00-461439ce7fef.png)
   ![image](https://user-images.githubusercontent.com/1246453/146310194-a22193ee-b62e-4810-bb5d-7359765af754.png)

**Comment by macsforme** [comment]
@allejo, can you verify whether this is still needed in light of #328?  The SDL2 framework was done away with and SDL is now statically linked, so hopefully that resolves this issue.

**Comment by allejo** [comment]
I can confirm that this issue is fixed by #328
